src/components/general/TextInput.svelte

```diff
@@ -3,7 +3,7 @@
 	export let inputPlaceholder;
 	export let textValue;
 
-	export let styleClass = 'input-accent';
+	const clazz = $$props.class || 'input-accent';
```

Comment:
would be better to have some speaking name and not a hack for the reserved class ;-)